python/golem_task_api/entrypoint.py

```diff
@@ -21,7 +25,16 @@
         requestor_lifecycle_handler: Optional[AppLifecycleHandler] = None,
         provider_handler: Optional[ProviderAppHandler] = None,
         provider_lifecycle_handler: Optional[AppLifecycleHandler] = None,
+        log_level: str = None,
 ):
+    level = log_level or logging.INFO
+    logging.basicConfig()
```

Comment:
I know it's a starting point, but I haven't seen a case when it wasn't quickly changed into `dictConfig()` :) So IMHO it's better to implement it in the beginning.

Reply:
great to prepare for the future, updated!